debugger/oe-gdb

```diff
@@ -3,5 +3,15 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-export PYTHONPATH=${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/openenclave/debugger/gdb-sgx-plugin
-LD_PRELOAD=${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/openenclave/debugger/liboe_ptrace.so gdb -iex "directory ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/openenclave/debugger/gdb-sgx-plugin" -iex "source ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/openenclave/debugger/gdb-sgx-plugin/gdb_sgx_plugin.py" -iex "set environment LD_PRELOAD" -iex "add-auto-load-safe-path /usr/lib" "$@"
+# Get path of the oe-gdb script
+OE_GDB_DIR=$(dirname "$0")
```

Comment:
[Danger!](https://mywiki.wooledge.org/BashFAQ/028)

Reply:
> A UNIX paradigm exists to solve this problem for you: configuration artifacts of your scripts should exist in either the user's HOME directory or /etc
This approach makes it difficult to have multiple versions of a script to coexist; pretty much precludes execution of a development version of a script. Is the development version of a script (~/work/oe-repo1/build/bin/foo.sh) supposed to overwrite the users HOME directory or /etc?

Reply:
It's terrible, isn't it?! If we're definitely using Bash, then I think the safest approach given the outlined  caveats is `BASH_SOURCE`.

Reply:
Ok. Let me try to use the BASH_SOURCE; I wish there was a simple solution.

Reply:
Me too.

Reply:
Fixed using BASH_SOURCE